### DISCUSSION TITLE
VCRUNTIME_1.dll was not found when i m installing in windows

### DISCUSSION THREAD

**Discussion by aman290204** [KRTirtho/spotube#1200] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

i cant open the appp

### Expected Behavior

it should open 

### Steps to reproduce

![image](https://github.com/KRTirtho/spotube/assets/158546069/fa71017f-1c7a-425e-9e04-df0c1806d35c)

### Operating System

windows 11

### Spotube version

latest

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Install [vcredist](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
